app/scripts/services/logout.js

```diff
@@ -0,0 +1,32 @@
+angular
+  .module('confRegistrationWebApp')
+  .service('logoutService', function(
+    $location,
+    $cookies,
+    $window,
+    $rootScope,
+    $sce,
+  ) {
+    this.logoutFormProviders = function(response) {
+      switch ($cookies.get('crsAuthProviderType')) {
+        case 'FACEBOOK':
+          // if facebook, then logout from facebook using url generated on the backend
+          if (response.data) {
+            $window.location.href = response.data.url;
+          }
+          break;
```

Comment:
You could destructure the response to just grab data and then make it an arrow function as well.
```suggestion
    this.logoutFormProviders = ({ data }) => {
      switch ($cookies.get('crsAuthProviderType')) {
        case 'FACEBOOK':
          // if facebook, then logout from facebook using url generated on the backend
          if (data) {
            $window.location.href = data.url;
          }
          break;
```

Reply:
done